webpack.browser.config.js

```diff
@@ -19,18 +22,34 @@ module.exports = {
 		],
 	},
 	plugins: [
-		new LodashModuleReplacementPlugin( {
-			paths: true,
-			shorthands: true,
-		} ),
 		new webpack.DefinePlugin( {
-			'process.env.NODE_ENV': JSON.stringify( 'production' ),
+			'process.env.NODE_ENV': JSON.stringify( env ),
 		} ),
-		new webpack.optimize.ModuleConcatenationPlugin(),
-		new UglifyJsPlugin(),
 	],
 	resolve: {
 		extensions: [ '.js', '.jsx' ],
 		modules: [ path.resolve( __dirname ), path.resolve( __dirname, 'node_modules' ) ],
 	},
 };
+
+switch ( env ) {
+	case 'development':
+		config.devtool = 'source-map';
+		config.devServer = {
```

Comment:
Do we use the development server in this case?

Reply:
I thought it might be useful, for whenever we try to embed HC client into a regular site to have a server that serves both js and css instead of publishing it somewhere.